<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCoordinateToKml</class>
 <widget class="QDialog" name="DialogCoordinateToKml">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>大地坐标转KML文件</string>
  </property>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>选择文件</string>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>521</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_24">
     <item>
      <widget class="QPushButton" name="pushButton_Tans_SelectFile">
       <property name="text">
        <string>打开文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Trans_FilePath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Tans_TansFile">
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>说明：坐标文件为逗号分隔符文件（CSV），格式为“点号,纬度,经度,椭球高,编码”，椭球高可省略。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>150</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>编码可省略，若按编码识别，编码为“0”时表示单点。</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>转换设置</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonPoint">
       <property name="text">
        <string>单点</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonLine">
       <property name="text">
        <string>连线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonCode">
       <property name="text">
        <string>按编码识别</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>170</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>除编码“0”以外，其它所有编码相同的点会按先后顺序连成一条线，并用编码标记线名。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
